lib/tasks/elasticsearch.rake

```diff
@@ -1,32 +1,18 @@
-require "elasticsearch/rails/tasks/import"
-
 namespace :elasticsearch do
-  task :drop do
-    Rubygem.__elasticsearch__.client.indices.delete index: Rubygem.index_name
+  task drop: :environment do
+    Rubygem.searchkick_index.delete
   end
-  task :create do
-    Rubygem.__elasticsearch__.create_index! force: true
-    Rubygem.__elasticsearch__.refresh_index!
+  task create: :environment do
+    Rubygem.searchkick_index.create
```

Comment:
`create` isn't really meant be to called directly (except for advanced use cases) - `reindex(import: false)` will ensure aliases are set up correctly.